sunpy/net/fido_factory.py

```diff
@@ -111,7 +70,9 @@ def __getitem__(self, aslice):
         returned from those clients.
         """
         # Just a single int as a slice, we are just indexing client.
-        if isinstance(aslice, (int, slice)):
+        if isinstance(aslice, int):
+            ret = [self._list[aslice]]
```

Comment:
Could instead do `aslice = slice(aslice, aslice + 1)` here, then remove the line below?